### DISCUSSION TITLE
[NodeJS]dependency grpc is old,need update

### DISCUSSION THREAD

**Discussion by alanlvle** [apache/skywalking#8119] [Ideas]
grpc npm homepage:
https://www.npmjs.com/package/grpc

info:
This library will not receive further updates other than security fixes. We recommend using @grpc/grpc-js instead

we company use @grpc/grpc-js,SkyWalking  NodeJS can update?

Tks

**Comment by wu-sheng** [comment]
You could send a pull request to update. We will help on reviewing.

**Comment by alanlvle** [comment]
i try to do it .
https://github.com/apache/skywalking-nodejs/pull/65
